internal/render/common/limits.go

```diff
@@ -0,0 +1,18 @@
+package common
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+func CopyNonCPULimits(resourceList corev1.ResourceList) corev1.ResourceList {
```

Comment:
What about naming it "ExcludeCPU" or "CopyNonCPUResources? "CopyLimits" is used a little bit wrongly here IMO, because it takes not "limits", but "resources".

The fact that excluding CPU from resources may be useful for setting container limits can be mentioned in the comment to this function (not in a comment inside the function body).